webapp/components/interop.html

```diff
@@ -269,16 +269,13 @@
       };
     }
 
-    async connectedCallback() {
-      await super.connectedCallback();
+    async ready() {
+      super.ready();
+
```

Comment:
I'm not familiar with `connectedCallback` (and why we don't need it anymore). Could you explain a bit or give me a pointer?